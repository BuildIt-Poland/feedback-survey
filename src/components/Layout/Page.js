import styled from 'styled-components';

import media from '../../styles/media';

const Page = styled.section`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  flex: 1;
  padding: 1rem;
  max-width: 107rem;

  ${media.mobileL`
    padding: 2rem;
  `}

  ${media.tablet`
    padding: 3rem 4rem 7rem;
  `}
`;

export default Page;
