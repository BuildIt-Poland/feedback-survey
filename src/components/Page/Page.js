import styled from 'styled-components';

const Page = styled.article`
  min-height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  padding: 5rem 4rem 2rem;
`;

export default Page;
