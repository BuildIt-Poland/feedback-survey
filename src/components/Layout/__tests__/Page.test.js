import React from 'react';
import { create } from 'react-test-renderer';

import Page from '../Page';

describe('COMPONENT - Layout Page', () => {
  it('renders correctly', () => {
    const component = create(<Page />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
