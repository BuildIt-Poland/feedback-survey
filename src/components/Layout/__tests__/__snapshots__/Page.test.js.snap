// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Layout Page renders correctly 1`] = `
<section
  className="sc-bdVaJa bBGsNk"
/>
`;
